spec/draper/decorator_spec.rb

```diff
@@ -664,6 +664,26 @@ def hello_world
           expect{decorator.hello_world}.to raise_error NoMethodError
           expect(decorator.methods).not_to include :hello_world
         end
+
+        context 'When decorator overrides a public method defined on the object with a private' do
+          let(:decorator_class) do
+            Class.new(Decorator) do
+              private
+
+              def hello_world
+                'hello world'
+              end
+            end
+          end
+
+          let(:object) { Class.new{ def hello_world; end }.new }
```

Comment:
Small spacing issue:

```suggestion
          let(:object) { Class.new { def hello_world; end }.new }
```